circe-checksum/pom.xml

```diff
@@ -48,6 +48,12 @@
       <version>${guava.version}</version>
     </dependency>
 
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-buffer</artifactId>
```

Comment:
For this artifact we need to be careful. Sure, the checksum only uses the `ByteBuf`, though in the other components we're already using the `netty-all` dependency which already includes the `netty-buffer` but maven will pull in both since it doesn't know it.

Reply:
@merlimat I exclude `netty-buffer` in bookkeeper-server when switching it from `pulsar-checksum` to use this imported module. (see the last commit in #937 )